docs/Contributing/architecture/mdm/disk-encryption.md

```
@@ -37,25 +35,207 @@ For macOS, disk encryption involves a two-step process:
 ## Key storage and security
 
-Encryption keys are stored in the `host_disk_encryption_keys` table. The value for the key is encrypted using Fleet's CA certificate, and thus can only be decrypted if you have the CA private key.
+Encryption keys are stored in the `host_disk_encryption_keys` table. The value for the key is
+encrypted using Fleet's CA certificate, and thus can only be decrypted if you have the CA private
+key. Additionally, a backup copy of any key that gets escrowed is stored in the
+`host_disk_encryption_keys_archive` table. In the event that a host's encryption key is unavailable
+due to a rotation or other event, it is possible to restore the host's most recently archived key by
+executing the following query againt the Fleet server's MySQL database, replacing HOST_ID with the
+ID of the host in question:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)